### PR TITLE
fix: table content active link not showing properly

### PR DESCRIPTION
**Pull request by ParasSolanki**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

In the create t3 app docs website the table of contents active link was not showing properly as you can see in first video.
Fixed that issue in second video.
Open for any feedback and improvements.

---

## Screenshots

Not working example video
[table-content-active-link-not-showing-properly.webm](https://user-images.githubusercontent.com/62164346/199393037-3f000112-b1f2-4595-9341-a4d526963c9c.webm)

Working video demo
[working-active-link-of-table-contents.webm](https://user-images.githubusercontent.com/62164346/199393798-2459ffba-fbdf-48a4-be87-2499434a75cb.webm)

💯

### CODE REVIEW

File: www/src/components/navigation/tableOfContents.astro [hunk 1]
```diff
@@ -50,19 +50,66 @@ headings = [{ depth: 2, slug: "overview", text: title }, ...headings].filter(
...
         entries.forEach((entry) => {
           const id = entry.target.id;
+          // do nothing if id is toc-heading
+          if (id === "toc-heading") return;
           // have to do "All" here cause we render this component in pageContent for mobile too
           const tocItem = Array.from(
             document.querySelectorAll(`a[href="#${id}"]`),
           ).at(-1);
           // this occurs when the id = "toc-heading"
           if (!tocItem) return;
```
Comment (c-ehrlich) [inline www/src/components/navigation/tableOfContents.astro:60]: do we still need this line with the check you added in line 54?

Comment (ParasSolanki) [reply www/src/components/navigation/tableOfContents.astro:60]: yeah we can remove it same as line 58 check

**Comment by juliusmarminge** [comment]
Astro's formatting tools in editors are a bit hit and miss, please run `pnpm format` to make sure they are formatted correctly.

**Comment by juliusmarminge** [comment]
Seems to not quite catch the `Useful Resources` table though but who cares, it's waaay better than before :)

https://create-t3-881fi3o7e-t3-oss.vercel.app/en/usage/prisma

CleanShot 2022-11-03 at 15 01 53@2x

**Review by juliusmarminge — APPROVED** [review]

**Comment by ParasSolanki** [comment]
Yeah if the last content is big enough that can reach at the top then it will get highlighted.

image

**Comment by juliusmarminge** [comment]
Yes i was just thinking if i see one header that should be the one which is highlighted but thats really not so dont mind me on that one

**Comment by ParasSolanki** [comment]
looks like on my PC its working maybe your PC resolution is big

image

**Comment by juliusmarminge** [comment]
> looks like on my PC its working maybe your PC resolution is big
> 
> 
> 
> image
> 
> 

MBP14 with More Space

**Comment by ParasSolanki** [comment]
Mine is smaller one Lenovo normal HD resolution.

**Comment by juliusmarminge** [comment]
Yeah the header is just down at the bottom for me xd 

https://share.cleanshot.com/swETEq

**Pull request merged**